front/cypress/e2e/admin/pages_and_menu/create_update_and_view_custom_page.cy.ts

```diff
@@ -138,6 +145,31 @@ describe('Admin: create, update, and view custom page', () => {
     //  go back to main page
     cy.get(`[data-cy="breadcrumbs-${pageName}"]`).click();
 
+    // turn off top info section again
+    cy.get(
+      `[data-cy="e2e-admin-section-toggle-top_info_section_enabled"]`
+    ).click();
+    // wait for the call to complete
+    cy.wait('@updateCustomPage');
+    // wait for the toggle to become enabled in the UI
+    cy.get(`[data-cy="e2e-admin-section-toggle-top_info_section_enabled"]`)
+      .find('i')
```

Comment:
What is i? Could you add a comment here?

Reply:
The toggle from our component library is a styled `i` tag: https://github.com/CitizenLabDotCo/cl2-component-library/blob/master/src/components/Toggle/index.tsx#L32